ci/tests/Jenkinsfile.e2e-upgrade

```diff
@@ -0,0 +1,84 @@
+pipeline {
+
+  agent { label 'linux' }
+
+  parameters {
+    string(
+      name: 'NETWORK',
+      description: 'Name of test network to use.',
+      defaultValue: 'ropsten',
+    )
+    string(
+      name: 'APK_NAME',
+      description: 'Filename of APK uploaded to SauceLabs',
+    )
+     string(
+      name: 'APK_NAME_UPGRADE',
+      description: 'Filename of APK of upgraded application',
```

Comment:
I don't get it. Is this the one that gets installed on top of the other or vice versa?

Reply:
APK for https://ci.status.im/job/end-to-end-tests/job/status-app-upgrade/ means release build (so base for upgrade)
I'll change the description, thanks